src/FiniteElementDiscretization.cxx

```diff
@@ -80,7 +80,7 @@ namespace mfem_mgis {
         params, FiniteElementDiscretization::NumberOfUniformRefinements, 0);
     if (parallel) {
 #ifdef MFEM_USE_MPI
-      auto smesh = std::make_shared<Mesh<false>>(mesh_file.c_str(), 1, 1);
+      auto smesh = loadMeshSequential(mesh_file.c_str(),1,1);
```

Comment:
remove the `c_str` call here which create a useless temporary `std::string`

Reply:
OK done.